int32 motor_r
int32 motor_l
float32[] sonor
float32[] line
float32[] accel
float32[] gyro
float32[] mag
float32 mileage
bool movestatus
